src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsModal.component.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import 'vs/css!./modal';
```

Comment:
This should be `notebookViewsModal.css` right?

`"Error: Could not find file:///D:/a/azuredatastudio/azuredatastudio/src/../out/sql/workbench/contrib/notebook/browser/notebookViews/modal.css or it was empty"`

Reply:
Oops. Thanks!